### PR TITLE
Merge Testing Into Develop

### PR DESCRIPTION
**Pull request by stravid87**
Before requesting a review of this Pull Request, I certify that I have tested the following items in the application and all have passed my local tests without any side effects:

- [x] Logout and login;
- [x] Navigation between pages;
- [x] Creation of a new prompt;
- [x] Creation of a new entry;
- [x] Like and Dislike button;
- [x] Share button;
- [x] Exclusion of the newly created prompt;
- [ ] Deletion of the newly created entry;

---

##DESCRIPTIONS
Here is the testing branch. It's now ready for review. The problem with deleting the entry persists but I don't want this to delay our use of the testing branch. As soon as we get real users (I've started to reach out), the development and feature requests are doing to come hard and fast. Too hard and too fast. So I feel that good test coverage, an robust CI/CD pipeline are necessary before we get real users.

@arnonrdp can you work your magic now to add the Github action?
@hermannleboss can you modify the Bargraph test suite to add the necessary props and then build out the tests
@NJavokhir can you use the test files present to start building tests for commenting?

Amazing guys! We're so close to entering the virtuous dev/ops cycle that EVERY Silicon Valley book ever talks about. All those start-ups in the Valley would be proud!

### CODE REVIEW

**Comment by arnonrdp** [comment]
Great to see the tests start to work.

Absolutely! I'll stick with the settings on GitHub Actions.

**Comment by arnonrdp** [comment]
@globe-and-citizen/team-1-test, everything seems to be working perfectly.

In my penultimate [commit](https://github.com/globe-and-citizen/Celebrity-Fanalyzer/pull/115/commits/a261739f8b5973498431fb084945df7e194422a2) I removed Hosting as I know it will not be used at this time. However, I wonder if there isn't anything else that can/should be removed.

For example, I've noticed that those test components have been removed, but I'm wondering if the [`/coverage`](https://github.com/globe-and-citizen/Celebrity-Fanalyzer/tree/testing_for_merge/coverage) folder is needed or if there's something else we don't need right now.

File: .gitignore [hunk 1]
```diff
@@ -20,6 +20,7 @@ node_modules
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+ui-debug.log
```
Comment (hermannleboss) [inline .gitignore:23]:
```suggestion
ui-debug.log
/coverage
```

Comment (hermannleboss) [reply .gitignore:23]: You don't need to add coverage output in git-versioned files.

**Review by hermannleboss — CHANGES_REQUESTED** [review]

**Comment by stravid87** [comment]
> @globe-and-citizen/team-1-test, everything seems to be working perfectly.
> 
> In my penultimate [commit](https://github.com/globe-and-citizen/Celebrity-Fanalyzer/pull/115/commits/a261739f8b5973498431fb084945df7e194422a2) I removed Hosting as I know it will not be used at this time. However, I wonder if there isn't anything else that can/should be removed.
> 
> For example, I've noticed that those test components have been removed, but I'm wondering if the [`/coverage`](https://github.com/globe-and-citizen/Celebrity-Fanalyzer/tree/testing_for_merge/coverage) folder is needed or if there's something else we don't need right now.

@hermannleboss I have not dived into Code Coverage yet. I remains on my ToDo. Hermann, what are you thoughts?

**Comment by hermannleboss** [comment]
I think we do not need it, because it will change depending on testing we create and it will create many files to track

**Review by stravid87 — COMMENTED** [review]
Simple change to make it work.

**Review by arnonrdp — APPROVED** [review]

**Pull request merged**